### ISSUE TITLE
positioning inaccurate on wide monitors

### ISSUE THREAD

**Issue by geddski** [pixijs/pixi-spine#248]
I'm seeing the strangest bug that only happens on certain browser window sizes. The coordinate system starts getting off. Using simple positioning like this:

```
this.character.skeleton.x = 100;
```

At widescreen the object shows up at the `100px` mark like it should. But if I make the browser window a little bit taller, the object shows up at `112px` instead. The taller the window, the more the character's positioning is off. 

What could cause this? 

I'm on the latest pixi and pixi-spine.

**Comment by geddski** [comment]
I should note that a Pixi.Graphic() doesn't suffer the same problem, only pixi-spine object.

**Comment by ivanpopelyshev** [comment]
Give me more details.

Also, look at https://github.com/pixijs/pixi.js/issues/4153

**Comment by geddski** [comment]
Yes that does seem to be the issue. At larger resolutions the size of my canvas and the drawingBuffer become out of sync. Do you know of a good solution for it?

**Comment by ivanpopelyshev** [comment]
Nope. Just the one posted there.

**Comment by geddski** [comment]
Example screen size with no bug:
canvas size: 7680 x 1976
drawingBuffer: 7680 x 1976

Example screen size **with** bug:
canvas size: 7680 x 3008
drawingBuffer: 6544 x 2563

It's like Pixi is limiting the drawingBuffer size.

**Comment by geddski** [comment]
changing my canvas size to be a fraction of the drawingBuffer size doesn't seem to fix it
